exchanges/binanceus/binanceus_types.go

```diff
@@ -0,0 +1,1074 @@
+	UpdateTime        time.Time `json:"updateTime"`
+	IsWorking         bool      `json:"isWorking"`
+	OrigQuoteOrderQty float64   `json:"origQuoteOrderQty"`
+}
+
+// OrderReportItem this is used by the OCO order creating response
+type OCOOrderReportItem struct {
+	CommonOrder
+	TransactionTime time.Time `json:"transactionTime"`
+}
+
+// GetOrderRequestParams this struct will be used to get a
+// order and its related information
+type OrderRequestParams struct {
+	Symbol            string `json:"symbol"` // REQUIRED
+	OrderID           uint64 `json:"orderId"`
+	OrigClientOrderID string `json:"origClientOrderId"`
+	recvWindow        uint
+}
+
+// CancelOrderRequestParams this struct will be used as a parameter for
+// cancel order method.
+type CancelOrderRequestParams struct {
+	Symbol            currency.Pair
+	OrderID           uint64
+	OrigClientOrderID string
+	NewClientOrderID  string
+	RecvWindow        uint
+}
+
+// GetTradesParams  request param to get the trade history
+type GetTradesParams struct {
+	Symbol     string     `json:"symbol"`
+	OrderID    uint64     `json:"orderId"`
+	StartTime  *time.Time `json:"startTime"`
+	EndTime    *time.Time `json:"endTime"`
+	FromID     uint64     `json:"fromId"`
+	Limit      uint       `json:"limit"`
+	RecvWindow uint       `json:"recvWindow"`
+}
+
+// Trade this struct represents a trade response.
+type Trade struct {
+	Symbol          string    `json:"symbol"`
+	ID              uint64    `json:"id"`
+	OrderID         uint64    `json:"orderId"`
+	OrderListID     int       `json:"orderListId"`
+	Price           float64   `json:"price"`
+	Qty             float64   `json:"qty"`
+	QuoteQty        float64   `json:"quoteQty"`
+	Commission      float64   `json:"commission"`
+	CommissionAsset float64   `json:"commissionAsset"`
+	Time            time.Time `json:"time"`
+	IsBuyer         bool      `json:"isBuyer"`
+	IsMaker         bool      `json:"isMaker"`
+	IsBestMatch     bool      `json:"isBestMatch"`
+}
+
+// OCOOrderParams
+// One -cancel-the-other order creation input Parameter
+type OCOOrderInputParams struct {
+	Symbol               string  `json:"symbol"`    // Required
+	StopPrice            float64 `json:"stopPrice"` // Required
+	Side                 string  `json:"side"`      // Required
+	Quantity             float64 `json:"quantity"`  // Required
+	Price                float64 `json:"price"`     // Required
+	ListClientOrderID    string  `json:"listClientOrderId"`
+	LimitClientOrderID   string  `json:"limitClientOrderId"`
+	LimitIcebergQty      float64 `json:"limitIcebergQty"`
+	StopClientOrderID    string  `json:"stopClientOrderId"`
+	StopLimitPrice       float64 `json:"stopLimitPrice"`
+	StopIcebergQty       float64 `json:"stopIcebergQty"`
+	StopLimitTimeInForce string  `json:"stopLimitTimeInForce"`
+	NewOrderRespType     string  `json:"newOrderRespType"`
+	RecvWindow           uint64  `json:"recvWindow"`
+}
+
+// GetOCOPrderRequestParams
+type GetOCOPrderRequestParams struct {
```

Comment:
```suggestion
// GetOCOOrderRequestParams ...
type GetOCOOrderRequestParams struct {
```
Please write comment also